2022q4/Makefile

```diff
@@ -35,7 +35,7 @@ Documentation=	doceng.adoc
 
 # Changes affecting the Ports Collection, whether sweeping changes that
 # touch most of the tree, or individual ports themselves
-Ports= FreshPorts.adoc portsdb.adoc xfce.adoc pantheon.adoc budgie.adoc gcc.adoc biology.adoc
+Ports= FreshPorts.adoc portsdb.adoc kde.adoc xfce.adoc pantheon.adoc budgie.adoc gcc.adoc biology.adoc
```

Comment:
should we maybe group desktop things?

pantheon, kde, xfce, etc

Reply:
I think it is better to avoid it. Desktop things live in the ports tree, so their actual categorization is correct. On the other hand, starting to create more specific categories of reports that are suited for the Ports category can quickly generate a lot of new categories; indeed the ports tree itself is subdivided in categories.

This is different than with the Cloud category: the cloud reports were put in the Projects category, which has a vague description; those reports were put in there mostly for lack of a better place. Moreover, the absence of a Cloud category was probably due to the fact that categories were first thought when the cloud was not so widely used as today: it makes sense to consider to create this category now that the cloud gets increasingly popular.